code/game/objects/items/storage/boxes.dm

```diff
@@ -1174,6 +1174,18 @@
 		new /obj/item/reagent_containers/food/snacks/grown/cabbage(src)
 	new /obj/item/reagent_containers/food/snacks/grown/chili(src)
 
+/obj/item/storage/box/ingredients/seafood
+	theme_name = "seafood"
+
+/obj/item/storage/box/ingredients/seafood/PopulateContents()
+	new /obj/item/reagent_containers/food/snacks/grown/citrus/lemon(src)
+	for(var/i in 1 to 6)
+		var/randomFood = pick(/obj/item/reagent_containers/food/snacks/carpmeat,
```

Comment:
you sure carpmeat is fine?

Reply:
Another box contains it, but if you're worried about having too much carpotoxin for free roundstart, I can limit it to 1 or 2, I don't mind. But you'd have to be insanely lucky currently to roll this box and then get extra carp meat and then it be on a round that needs it.